src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
+import style from './style.scss';
+import numberInRange from '../../utils/numberInRange';
+
+const classNames = cx.bind(style);
+const colorMap = {
+  success: 'color-success',
+  warning: 'color-warning',
+  error: 'color-error',
+};
+
+const Progressbar = ({ percentage = 0, showPercentageNum, className, color, ...props }) => {
+  const progressBarWidth = percentage > 100 ? 100 : percentage;
+  const percentageLabel = showPercentageNum ? `${percentage}%` : null;
+  const progressBarColor = colorMap[color] || 'color-success';
+  const dynamicStyle = {
+    width: `${progressBarWidth}%`,
+  };
+
+  return (
+    <div className={classNames('Progressbar')} {...props}>
+      <div style={dynamicStyle} className={classNames('Progressbar_progress', progressBarColor, className)}>
+        {percentageLabel}
+      </div>
+    </div>
+  )
+};
+
+Progressbar.propTypes = {
+  /** Custom class to percentage. Valid range is \[0, 100\] */
+  percentage: numberInRange(0, 100),
+  showPercentageNum: PropTypes.bool,
+  className: PropTypes.string,
+  /** Color options include \['success', 'warning', 'error'\] */
+  color: PropTypes.string,
+};
+
+Progressbar.defaultProps = {
+  percentage: 0,
+  showPercentageNum: false,
+  color: 'success',
```

Comment:
```jsx
color: 'primary'
```

Reply:
Also, can we have `isFlat` option? if `isFlat` is true, then no radius, if it is false, then same as now. What do you think?